### DISCUSSION TITLE
Clarify stance on i18n in app directory

### DISCUSSION THREAD

**Discussion by dotellie** [vercel/next.js#44679] [Help]
There has been a lot of discussion about i18n in the app directory where the stance from the next team seems to have been that it's better to have applications implement it separately and we have seen a few examples of this from various sources. Me personally, I don't exactly agree with the stance but I can see the reasoning behind it. For me it just seems that not supporting i18n out of the box will add a bunch of boilerplate code to any app using i18n which seems like a shame.

Anyway, my question (and confusion) here is what the stance really is. Previously there was a line in the docs stating that i18n would not be implemented for the app directory but now, as far as I can see, that line is gone. So, to get to the point, **is i18n going to be implemented into the app directory or not**?

_Screenshot of where it was previously stated i18n would not be implemented:_
<img width="713" alt="image" src="https://user-images.githubusercontent.com/4590037/211143665-f477efb2-1e78-4a1e-abe5-ac1f4becf202.png">

**Comment by leerob** [comment]
We've published documentation on Internationalization (i18n). Please leave feedback on the page! 🙏

https://beta.nextjs.org/docs/guides/internationalization

**Reply by dotellie** [reply]
That's great, thanks! I would however still put back the notice on the documentation page considering that the feature isn't actually implemented in Next.js (app directory) and, from what I can tell from your answer, isn't planned to be implemented. It would probably be good to still have up as a "gotcha" considering routing still has quite a lot of localization features still built in that won't work with the method described in that example documentation.
